### PR TITLE
Templates tips and tricks

### PR DESCRIPTION
**Pull request by klox7**
Examples of template modifications.

### CODE REVIEW

**Comment by Serhioromano** [comment]
Hi. I am sorry I did not see this. Could you please apply few changes.

1. In the head of the document add something liek this

```
---
layout: doc
title:  "Templates tips and tricks"
date:   2015-02-15 12:30:30
tags: templates
intro: "Additional information."
---
```

2. Rename file to `2015-02-15-template-tips-and-tricks`

**Comment by klox7** [comment]
Done. I hope it's OK. You can also use this for [comments with replies!](https://github.com/klox7/cobalt/blob/master/comments%20and%20reviews.md) if you want.

File: en/cobalt/_posts/user/2015-02-15-template-tips-and-tricks [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+## Markup template
+
+You can find markup template in *components/com_cobalt/views/records/tmpl/default_markup_default.php*
+
+#### Remove dropdown Switch view and change template labels to images with labels as tooltips
+
+So let's try do change this template from
+
+![Dropdown Switch view markup template](http://demo.web-er.si/markup_template_template_switcher_before.jpg)
```
Comment (Serhioromano) [inline en/cobalt/_posts/user/2015-02-15-template-tips-and-tricks]: you use image here and also later. Please store screenshot in `assets/img/screenshots` and then use something like this

    Dropdown Switch view markup template

This way we ensure screenshot availability.

**Comment by Serhioromano** [comment]
Another change file have to have extension `.md`

I appologize for a lot of comments.

**Comment by Serhioromano** [comment]
I'll ad changes myself.

**Pull request merged**